docs/setup-vendor.md

```diff
@@ -0,0 +1,47 @@
+# Getting started with Private Packagist for Vendors in a Composer project
+
+Using Private Packagist for Vendors, customers install packages with Composer using a unique URL and an authentication token.
+
+Customers can start using Private Packagist in either an existing or a new Composer project.
+
+## Adding a Package from a vendor to an existing Composer project
+
+To install a package named "acme-company/api" into your existing project, you need to follow the steps below:
```

Comment:
"You" in this case is the vendor, not the customer, I'll stop the review here. Please re-read the article, make sure the entire article is directed at vendors, not customers, then ping me to review again, thanks!

Reply:
ok updated and made sure it's directed towards vendors.